<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1018</width>
         <height>626</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Canvas" name="canvas" native="true">
          <property name="minimumSize">
           <size>
            <width>1000</width>
            <height>500</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionBy_ID"/>
    <addaction name="actionBy_Area"/>
    <addaction name="actionBy_Perimeter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuReports"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Add Shapes</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Ellipse"/>
   <addaction name="actionAdd_Line"/>
   <addaction name="actionAdd_Polygon"/>
   <addaction name="actionAdd_Polyline"/>
   <addaction name="actionAdd_Rectangle"/>
   <addaction name="actionAdd_Text"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="PropDock">
   <property name="windowTitle">
    <string>Property Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shapes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ShapeList"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Properties</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="PropTree">
       <property name="minimumSize">
        <size>
         <width>268</width>
         <height>0</height>
        </size>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove Shape</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Ellipse">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/add_ellipse.jpg</normaloff>
     <normalon>:/icons/add_ellipse.png</normalon>:/icons/add_ellipse.jpg</iconset>
   </property>
   <property name="text">
    <string>Add Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Add an ellipse</string>
   </property>
   <property name="statusTip">
    <string>Add an ellipse</string>
   </property>
  </action>
  <action name="actionAdd_Line">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/add_line.jpg</normaloff>:/icons/add_line.jpg</iconset>
   </property>
   <property name="text">
    <string>Add Line</string>
   </property>
   <property name="toolTip">
    <string>Add a line</string>
   </property>
   <property name="statusTip">
    <string>Add a line</string>
   </property>
  </action>
  <action name="actionAdd_Polygon">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/add_polygon.jpg</normaloff>:/icons/add_polygon.jpg</iconset>
   </property>
   <property name="text">
    <string>Add Polygon</string>
   </property>
   <property name="statusTip">
    <string>Add a polygon</string>
   </property>
  </action>
  <action name="actionAdd_Polyline">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/add_polyline.jpg</normaloff>:/icons/add_polyline.jpg</iconset>
   </property>
   <property name="text">
    <string>Add Polyline</string>
   </property>
   <property name="statusTip">
    <string>Add a polyline</string>
   </property>
  </action>
  <action name="actionAdd_Rectangle">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/add_rectangle.jpg</normaloff>:/icons/add_rectangle.jpg</iconset>
   </property>
   <property name="text">
    <string>Add Rectangle</string>
   </property>
   <property name="statusTip">
    <string>Add a rectangle</string>
   </property>
  </action>
  <action name="actionAdd_Text">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/add_text.jpg</normaloff>:/icons/add_text.jpg</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
   <property name="statusTip">
    <string>Add a text box</string>
   </property>
  </action>
  <action name="actionBy_ID">
   <property name="text">
    <string>By ID</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBy_Area">
   <property name="text">
    <string>By Area</string>
   </property>
  </action>
  <action name="actionBy_Perimeter">
   <property name="text">
    <string>By Perimeter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header location="global">canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
